pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -315,8 +315,16 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 }
 
 func updateStatusFromBuildStatus(taskRun *v1alpha1.TaskRun, buildStatus buildv1alpha1.BuildStatus) {
-	if buildStatus.GetCondition(duckv1alpha1.ConditionSucceeded) != nil {
+	if bs := buildStatus.GetCondition(duckv1alpha1.ConditionSucceeded); bs != nil {
+		// Build Failure
```

Comment:
`bs.IsFalse()` checks for bs != nil  this could become 

```
 buildStatus.GetCondition(duckv1alpha1.ConditionSucceeded).IsFalse()
```

Reply:
I'd like to keep it as is, in order to know if it's nil. Check the comment a few lines below.

> // If the buildStatus doesn't exist yet, it's because we just started running